resources/js/app.js

```diff
@@ -386,6 +386,16 @@
                 self.mentorsList = {}
                 self.adminsList = {}
 
+                $scope.getGsocYear = function() {
+                    var gsocSwitchMonth = 6;
```

Comment:
I prefer reading the number from https://github.com/coala/projects/blob/ca8a58fd7e914c63f6feb3a1519c98d32ccb942a/_config.yml#L9

Reply:
I could add frontmatter to the file `app.js` and use the variable as `var gsocSwitchMonth = {{ site.gsoc_switch_month }}`. Does this look like a good enough solution to you?

Reply:
Hi @li-boxuan Made the desired changes.